protected/views/site/login.php

```diff
@@ -1,69 +1,79 @@
 <?php
-$this->pageTitle=Yii::app()->name . ' - Login';
-$this->breadcrumbs=array(
-	'Login',
+$this->pageTitle = Yii::app()->name . ' - Login';
+$this->breadcrumbs = array(
+    'Login',
 );
 ?>
-<section>   
 
-<div class="container" id="login">
-   <section class="page-title-section">
-       <div class="page-title">
-            <ol class="breadcrumb pull-right">
-                <li><a href="/">Home</a></li>
-                <li class="active">login</li>
-            </ol>
-            <h4>Login</h4>
-       </div>
-   </section>
-	<div class="subsection row" style="margin-bottom: 130px;">
-        <div class="col-xs-12">
-            <?php if(Yii::app()->user->hasFlash('success-reset-password')): ?>
-                <div class="alert alert-success">
-                    <?php echo Yii::app()->user->getFlash('success-reset-password'); ?>
-                </div>
-            <?php endif; ?>
-        </div>
+<section>
+    <div class="container" id="login">
+        <section class="page-title-section">
+            <div class="page-title">
+                <ol class="breadcrumb pull-right">
+                    <li><a href="/">Home</a></li>
+                    <li class="active">login</li>
+                </ol>
+                <h1 class="h4">Login</h1>
+            </div>
+        </section>
+        <div class="subsection row" style="margin-bottom: 130px;">
+            <div class="col-xs-12">
+                <?php if (Yii::app()->user->hasFlash('success-reset-password')) : ?>
+                    <div class="alert alert-success">
+                        <?php echo Yii::app()->user->getFlash('success-reset-password'); ?>
+                    </div>
+                <?php endif; ?>
+            </div>
             <div class="col-xs-6">
                 <div class="subsection-login">
-                    <p><?=Yii::t('app' , 'Please fill out the following form with your login credentials:')?></p>
-                    <p><?=Yii::t('app' , 'Fields with <span class="symbol">*</span> are required.')?></p>
+                    <p><?= Yii::t('app', 'Please fill out the following form with your login credentials:') ?></p>
+                    <!-- inputs are already announced as required so screen readers do not need this message -->
+                    <p aria-hidden="true"><?= Yii::t('app', 'Fields with <span class="symbol">*</span> are required.') ?></p>
```

Comment:
Added aria-hidden="true" so that this text is not readable by screen readers. the reasoning is that inputs are already announced as required and the asterisk is purely visual